tso-server/src/main/resources/default-omid-server-configuration.yml

```diff
@@ -144,5 +144,4 @@ metrics: !!org.apache.omid.metrics.NullMetricsProvider [ ]
 #     zkNamespace: "omid"
 # metrics: !!org.apache.omid.metrics.NullMetricsProvider [ ]
 
-
-
+waitStrategy: HIGH_THROUGHPUT
```

Comment:
I think we should move this option to the line 16 (after the port config) as is one of they key config options now. Add also a comment to help users to use it:

```
# Wait strategy for the Disruptor processors in TSO pipeline. Options:
# 1) HIGH_THROUGHPUT - [Default] Use this in production deployments for maximum performance
# 2) LOW_CPU - Use this option when testing or in deployments where saving CPU cycles is more important than throughput
waitStrategy: HIGH_THROUGHPUT
```